### DISCUSSION TITLE
FOLLOW UP ON NEAR MOBILE WALLET

### DISCUSSION THREAD

**Discussion by kcelestinomaria** [near/near-wallet#1685] [Ideas]
Also, I got this idea for the mobile wallet, we could attach the public key to a mobile phone number, this will make it very easy to onboard users for dapps especially users in the third-world who mostly use mobile phones. I am building a product on NEAR targeted for these users. This will help NEAR a lot in scaling in the third-world, we really need a mobile version of this wallet. Thank you.

**Comment by stefanopepe** [comment]
I agree with the concept, although it may introduce some privacy issues: the people who know your phone number may be adversarial and use this information to target your NEAR wealth.
